contracts/loot/item/constants.cairo

```diff
@@ -6,54 +6,257 @@
 
 %lang starknet
 
+namespace Material:
+    const generic = 0
+
+    namespace Metal:
+        const generic = 0
+        const ancient = 1
+        const holy = 2
+        const ornate = 3
+        const gold = 4
+        const silver = 5
+        const bronze = 6
+        const platinum = 7
+        const titanium = 8
+        const steel = 9
+    end
+    namespace Cloth:
+        const generic = 0
+        const royal = 1
+        const divine = 2
+        const brightsilk = 3 
+        const silk = 4
+        const wool = 5
+        const linen = 6
+    end
+    namespace Biotic:
+        namespace Demon:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Dragon:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const skin = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Animal:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+        namespace Human:
+            const generic = 0
+            const blood = 1
+            const bones = 2
+            const brain = 3
+            const eyes = 4
+            const hide = 5
+            const flesh = 6
+            const hair = 7
+            const heart = 8
+            const entrails = 9
+            const hands = 10
+            const feet = 11
+        end
+    end
+    namespace Paper:
+        const generic = 0
+        const magical = 1
+    end
+    namespace Wood:
```

Comment:
@loothero Should we add `hickory` and `bamboo`? Some ideas:

http://www.aikiweb.com/weapons/goedkoop1.html
https://tanren-dojo.tripod.com/wood.html
https://ibook.pub/wood-use-in-early-medieval-weapon-production.html